src/components/FlawForm.vue

```diff
@@ -74,13 +75,19 @@ const {
 } = useFlawModel(props.flaw, onSaveSuccess);
 
 const initialFlaw = ref<ZodFlawType>();
+const { draftFlaw } = useDraftFlawStore();
 
 onMounted(() => {
   initialFlaw.value = deepCopyFromRaw(props.flaw) as ZodFlawType;
+  if(draftFlaw) {
+    flaw.value = useDraftFlawStore().addDraftFields(flaw.value);
+    useDraftFlawStore().$reset();
+  }
 });
 
 watch(() => props.flaw, () => { // Shallow watch so as to avoid reseting on any change (though that shouldn't happen)
   initialFlaw.value = deepCopyFromRaw(props.flaw) as ZodFlawType;
+  console.log(1,useDraftFlawStore().draftFlaw);
```

Comment:
Leftover console.log?

Reply:
Oh that was a catch from David yesterday I supposedly removed already

Reply:
Ah, sometimes going commit by commit is not ideal :smile: